extmod/vfs.c

```diff
@@ -360,7 +360,7 @@ mp_obj_t mp_vfs_getcwd(void) {
     if (!(cwd[0] == '/' && cwd[1] == 0)) {
         vstr_add_str(&vstr, cwd);
     }
-    return mp_obj_new_str_from_vstr(&mp_type_str, &vstr);
+    return mp_obj_new_str_from_vstr(&vstr);
```

Comment:
Can a filesystem have invalid chars in it?  How do you get the cwd if it does?  How does CPython handle this?